CesiumGeospatial/include/CesiumGeospatial/GlobeRectangle.h

```diff
@@ -61,6 +61,21 @@ class CESIUMGEOSPATIAL_API GlobeRectangle final {
       double north) noexcept
       : _west(west), _south(south), _east(east), _north(north) {}
 
+  /**
+   * @brief Constructs a new `GlobeRectangle` from the provided `Rectangle`.
+   *
+   * The rectangle's minimum coordinates will be interpreted as the southwest
+   * position and the maximum coordinates as the northeast.
+   *
+   * @param rectangle The rectangle defining the bounds of the new
+   * `GlobeRectangle`.
+   */
+  constexpr GlobeRectangle(const CesiumGeometry::Rectangle& rectangle) noexcept
```

Comment:
I think this implicit conversion is dangerous. It means that a `Rectangle` will be silently turned into a `GlobeRectangle`, but that only works when the original `Rectangle` is expressed in radians, which is not guaranteed. In other cases, it will silently produce wrong results. Better to leave this as an explicit conversion, even if it's more verbose.